Azure-Samples/azure-search-power-skills
# Fix TextQualityWatchdog Directory Structure for Azure Functions

## Purpose

* This PR fixes the project directory structure so that Azure Functions recognizes Watchdog as a function. This ensures that it can be recognized as an HTTP endpoint.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone https://github.com/WillChen789/azure-search-power-skills.git
```

* Test the code

```
After cloning the repo, open the TextQualityWatchdog directory in VS Code. Press F5. Azure Functions plugin should have a dropdown that has the "Watchdog" function at the very bottom. Right click and "execute now". Passing in the JSON from sample.dat should not raise any errors.
```

## What to Check
Verify that the following are valid
* All the files from the original PR are still working

## Other Information

None
